actors/evm/src/interpreter/instructions/hash.rs

```diff
@@ -26,3 +26,65 @@ pub fn keccak256(
 
     Ok(U256::from_big_endian(&buf[..size]))
 }
+
+#[cfg(test)]
+mod test {
+    use fil_actors_runtime::runtime::Primitives;
+    use rand::{distributions::Standard, Rng};
+
+    use crate::{evm_unit_test, interpreter::U256, BytecodeHash};
+
+    #[test]
+    fn keccak256_rand() {
+        let r = &mut rand::thread_rng();
```

Comment:
I guess I missed this in the previous tests...

Avoid randomness in tests. It makes them very difficult to debug if something actually goes wrong (i.e., you get flaky tests). In general, it's best to just use "deterministic" randomness based on a fixed seed.

However, in _this_ case, there's likely need for randomness at all. All we need to test are a few buffers of different sizes (e.g., 0, 1, 10, 100 bytes). That should cover any obvious error cases.

Anything _else_ would require continuous fuzzing, which is out of scope here.